src/rcpp_constants.cpp

```diff
@@ -1099,6 +1099,15 @@ String NPP_FLUX0(String biome = "") {
   return out;
 }
 
+
+//' @rdname carboncycle
+//' @export
+// [[Rcpp::export]]
+String PF_SIGMA() {
+    return D_PF_SIGMA;
+}
+
+
```

Comment:
PF_SIGMA() is the function that we use in R and it will return the string C++ knows. What I am not sure is if this needs to have any of the biome code like the `PERMAFROST_C`.